src/OpenAIGym.jl

```diff
@@ -37,51 +38,17 @@ GymEnv(name) = gym(name)
 function Reinforce.reset!(env::GymEnv)
     env.state = env.pyenv[:reset]()
     env.reward = 0.0
+    env.total_reward = 0.0
     env.actions = actions(env, nothing)
     env.done = false
 end
 
-"A simple wrapper around the OpenAI gym environments to add to the Reinforce framework"
-type UniverseEnv <: AbstractGymEnv
```

Comment:
I think Universe is basically dead, see here: https://github.com/openai/universe/issues/218#issuecomment-361711296 and the link there to this reddit comment by Ilya Sutskever here: https://www.reddit.com/r/MachineLearning/comments/6jjrk3/d_why_im_remaking_openai_universe/djfbk1b/

The Universe code being in the same file as the Gym code caught me out a couple of times when I edited the method for Universe instead of Gym, so if you really want to keep it I think it should be in a different file

Reply:
Okay, deleting it is fine at this moment.
Is this the replacement? https://github.com/openai/retro

Reply:
No that's essentially just more gym environments (though would need a very small amount of setup code to get running in julia I think). The guy from that github comment has this: https://github.com/unixpickle/muniverse and said he's working at OpenAI, but nothing official has been released AFAIK